test/facets/Token/behaviors/bridgeToken.t.sol

```diff
@@ -0,0 +1,290 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.19;
+
+import { TokenBridge } from "../TokenBridge.t.sol";
+import { ArbForkTest } from "../../../ArbForkTest.t.sol";
+import { ITokenBridge } from "../../../../contracts/facets/Token/ITokenBridge.sol";
+import { ITokenBridgeInternal } from "../../../../contracts/facets/Token/ITokenBridgeInternal.sol";
+import { IAxelarGasService } from "@axelar/interfaces/IAxelarGasService.sol";
+import { IAxelarGateway } from "@axelar/interfaces/IAxelarGateway.sol";
+
+/// @title Token_bridgeToken
+/// @notice This contract tests the functionalities of the bridgeToken function
+/// @dev It inherits from the TokenWithBridge contract and the ArbForkTest contract
```

Comment:
What is the `TokenWithBridge` contract?

Reply:
fixed here (43cafc686e37037c7a8a8713a593cf0ac7503b58)